fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -128,6 +123,22 @@ public AuthorizeController(
         .orElseThrow(NotFoundException::new);
   }
 
```

Comment:
I added these below since we didn't have them

Reply:
I'd prefer to just leave these out - we don't seem to have a need for them anywhere yet